### PR TITLE
fix: Fix chunked segment can not warmup using mmap

### PR DESCRIPTION
**Pull request by sunby**
issue: #38410

### CODE REVIEW

**Comment by czs007** [comment]
https://github.com/milvus-io/milvus/issues/38213

File: internal/core/src/storage/ChunkCache.cpp [hunk 1]
```diff
@@ -66,8 +71,28 @@ ChunkCache::Read(const std::string& filepath,
...
+        std::shared_ptr<Chunk> chunk;
+        auto dim = IsSparseFloatVectorDataType(field_meta.get_data_type())
+                       ? 1
+                       : field_meta.get_dim();
+        if (mmap_enabled) {
+            auto path = std::filesystem::path(CachePath(filepath));
+            auto dir = path.parent_path();
+            std::filesystem::create_directories(dir);
+
+            auto file = File::Open(path.string(), O_CREAT | O_TRUNC | O_RDWR);
```
Comment (bigsheeper) [inline internal/core/src/storage/ChunkCache.cpp:83]: Use mmap manager?

**Comment by wangting0128** [comment]
### verify panic
image: PR-38492-20241216-0bba7866

File: internal/core/src/storage/ChunkCache.cpp (hunk 1 above)
Comment (sunby) [reply internal/core/src/storage/ChunkCache.cpp:83]: > Use mmap manager?

Chunk and ChunkedColumn will be changed a lot if I use mmap here. Will change it to MmapChunkManager later.

**Pull request merged**